src/components/SideNavBar/index.tsx

```diff
@@ -1,10 +1,10 @@
 import React, { useRef, useState } from 'react';
-import { AboutPageSideNavBarItem } from '../../types';
+import { SideNavBarItem } from '../../types';
```

Comment:
Any specific reason for renaming these?

Since we're using the same sidebar for package-manager page as well, I think you could revert these changes.

Reply:
I thought about reverting, but I think I kinda like it. Its a side nav bar component that contains links to a myriad of pages, not just related links from the about page. A generic name is appropriate here imo

Reply:
@benhalverson What are your thoughts?

Reply:
Before we added package manager I think `AboutPageSideNavBarItem` made sense because it was only used on the about page.

I think `SideNavBarItem` fits better now if we plan to use it in multiple locations.